### DISCUSSION TITLE
how do i passthrough my camera on waydroid?

### DISCUSSION THREAD

**Discussion by ArefinKarim** [waydroid/waydroid#1248] [General]
My system specs:
Lenovo Thinkpad E490
System: Arch Linux x86_64
CPU: Intel i5-8265U (8) @ 3.900GHz
GPU: Intel WhiskeyLake-U GT2 [UHD Graphics 620]
Kernel: 6.7.0-zen3-1-zen
Memory: 7.6 GiB of RAM
DE: KDE Plasma 5.27.10
Graphics Platform: Wayland
Webcam as of lsusb: Bus 001 Device 003: ID 5986:2113 Bison Electronics Inc. SunplusIT Integrated Camera

Anyone know how do I passthrough my laptop built-in webcam on waydroid? (Sorry for my bad english)
Or anyway so that  I can add external camera?

**Comment by tsilvs** [comment]
Wait for this issue to be resolved: #519

**Reply by tomleejumah** [reply]
this has never been resolved?

**Reply by tsilvs** [reply]
Apparently. I am not following it nowadays.

**Reply by tomleejumah** [reply]
https://github.com/waydroid/android_external_minigbm/issues/3
